transformers/shared/constants/keys.go

```diff
@@ -45,43 +45,28 @@ const (
 	ArtColumn       event.ColumnName = "art"
 	BidColumn       event.ColumnName = "bid"
 	BidIDColumn     event.ColumnName = "bid_id"
-	BuyAmtColumn    event.ColumnName = "buy_amt"
-	BuyGemColumn    event.ColumnName = "buy_gem"
 	CdpColumn       event.ColumnName = "cdp"
 	DartColumn      event.ColumnName = "dart"
 	DataColumn      event.ColumnName = "data"
 	DinkColumn      event.ColumnName = "dink"
-	DustColumn      event.ColumnName = "dust"
```

Comment:
Is this only used in the Oasis transformers? I'm just thinking there's a vat_file `what = "dust"`, but maybe there aren't any `event.ColumnName ` entries for dust?

Reply:
I believe this file should only need to contain columns that are used in the oasis transformers (and their schema). I would have thought that we could also get rid of `dink`/`dart` and some of the other column names in this file

Reply:
Whoops I totally misunderstood this - we need `dink`/`dart` because this is MCD not Oasis 🤦 

I'm assuming the `dust` column wasn't actually used anywhere (as you suggested, since it's just a `what` value that gets parsed as a string)